README.rst

```diff
@@ -190,7 +190,7 @@ Update python dependencies
 
 ::
 
-  # from scriptworker-scripts/ ; this will run docker for py38 and py39
+  # from scriptworker-scripts/ ; this will run docker for py38 and py311
```

Comment:
Is python3.8 still needed because of iscript, or some other reason?

Reply:
iscript is still on 3.8, yes.

Reply:
Sadly 3.8 still.
If the move to Autograph + rcodesign fails for some reason we should look into upgrading the mac signers (os + python versions)